scripts/validate-pr.sh

```diff
@@ -32,6 +32,20 @@ main() {
         echo "PR title must start with '[$changed] '." >&2
         exit 1
     fi
+
+    if [[ -f charts/$changed/README.md ]]; then
+      cd charts/"$changed"
+      helm-docs
+      if [[ $(git diff --stat) != '' ]]; then
+        echo "Readme for chart wasn't updated. Please use helm-docs and update it"
+        echo "Diff:"
+        git diff
+        exit 1
+      else
+        echo 'clean'
```

Comment:
I doubt that this echo brings any value, either remove or change it to something like: "helm-docs generation check: OK"